src/analyze/analyze-security.c

```diff
@@ -1527,7 +1527,9 @@ static const struct security_assessor security_assessor_table[] = {
         },
 };
 
-static int assess(const SecurityInfo *info, Table *overview_table, AnalyzeSecurityFlags flags, unsigned threshold) {
+static int assess(const SecurityInfo *info, Table *overview_table, AnalyzeSecurityFlags flags,
+                  unsigned threshold, JsonVariant *v) {
```

Comment:
this is not how we usually line break long function lines. please place one argument per line, as we usually do